class/Contact.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+class Contact
+{
+    private $contactTable = 'contact';
```

Comment:
Change the variable name from $contactTable to $table. $table is more appropriate because it explains that the name of the table is contact.

Reply:
Also rename the table from contact to contacts. Table names should be pluralised cause it contains multiple rows of records/information.